include/envoy/router/router.h

```diff
@@ -11,6 +11,7 @@
 #include "envoy/common/optional.h"
 #include "envoy/http/codec.h"
 #include "envoy/http/header_map.h"
+#include "envoy/tracing/http_tracer.h"
```

Comment:
why is this included?

Reply:
Its required for the `apply(Tracing::Span&)` on `Decorator`.